app/models/application_record.rb

```diff
@@ -137,4 +147,29 @@ def self.ethnicity_eeo_data(records)
       [ethnicity, specific_records.count, specific_records.interview_count]
     end
   end
+
+  def self.notification_emails(records)
+    emails = Hash.new { |h, k| h[k] = {} }
```

Comment:
If you're going to call a method including the word `build` on it, seems weird to have to initialize first.